webapp/src/index.tsx

```diff
@@ -21,7 +21,7 @@ import SubscribeModal from 'containers/modals/SubscribeModal';
 import App from './app';
 
 // eslint-disable-next-line import/no-unresolved
-import {PluginRegistry} from './types/mattermost-webapp';
+import {PluginRegistry} from './types/mattermostWebapp';
```

Comment:
I think we should keep this folder as it is `mattermost-webapp` as it comes the plugin starter template.

Reply:
yes but I think to maintain consistency we should keep it the way we have kept others, otherwise keeping two patterns in the same repo will introduce conflicts